content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,158 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
+
+Selecting files with input elements or dropping them over the page is available
+in Cypress 9.3. Read the [`.selectFile()` API docs](/api/commands/selectfile)
+for more information on how this works and how to use it. This guide details how
+to change your test code to migrate from
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload) to
+`.selectFile()`.
+
+#### Quick guide
+
+The argument signature is different for Cypress' builtin `.selectFile()` command
+than the `.attachFile` command the `cypress-file-upload` plugin provided. You
+can follow the steps below for each argument in order to migrate:
+
+In the first argument:
+
+- `filePath`: Prefix value with `cypress/fixtures/`. Rename to `contents`.
+- `fileContent`: Use `Buffer.from()` rather than `Cypress.Blob` methods. Rename
```

Comment:
Do we need to explain to the user where `Buffer` is coming from here? Are they going to be confused?

Reply:
I've added a link to the nodejs docs for Buffer. Questions like "why can I use Buffer in my cypress code, I thought it executed in the browser" are beyond the scope of a migration guide.